src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -32,10 +32,12 @@ object ClusterStatus extends Enumeration {
   val Unknown, Creating, Running, Updating, Error, Deleting, Deleted = Value
   val activeStatuses = Set(Unknown, Creating, Running, Updating)
   val monitoredStatuses = Set(Unknown, Creating, Updating, Deleting)
+  val existingStatuses = Set(Unknown, Creating, Running, Updating, Error, Deleting)
```

Comment:
I see what you're going for here, but I think this needs a better name. `aliveStatuses`? Open to suggestions...

Reply:
`visibleStatuses`?